src/module.js

```diff
@@ -524,21 +524,27 @@ class DiscretePanelCtrl extends CanvasPanelCtrl {
       _.reverse(items);
     }
     
+    var singleMode = (items.length == 1);
+
     _.each(items, (point, i) => {
 
       var from = point.start;
       var to = point.start + point.ms;
       var time = point.ms;
       var val = point.val;
-      var seriesName = this.data[i].name;
+      if(!singleMode) {
+        var seriesName = this.data[i].name;
+      } else {
+        var seriesName = this.data[selectedIndex].name;
+      }
 
       var color = this.getColor(val);
 
       body += `
       <div 
         class="
           graph-tooltip-list-item 
-          ${i == selectedIndex ? 'graph-tooltip-list-item--highlight' : ''}
+          ${(i == selectedIndex) || (singleMode) ? 'graph-tooltip-list-item--highlight' : ''}
```

Comment:
At single mode you have just 1 item, therefore tooltip line was highlighted only at 1st row.

Reply:
no need to surround with `(`. So `(singleMode)` -> `singleMode`